plugin.xml

```diff
@@ -65,7 +61,14 @@
         </config-file>
 
 <!--        <framework src="SSZipArchive" type="podspec" spec="~> 2.2.3" />-->
-        <pod name="SSZipArchive" git="https://github.com/kitolog/ZipArchive" branch="master" />
+        <podspec>
+            <config>
+                <source url="https://github.com/CocoaPods/Specs.git" />
+            </config>
+            <pods use-frameworks="true">
+                <pod name="SSZipArchive" git="https://github.com/kitolog/ZipArchive" branch="master" />
```

Comment:
please specify current `commit` of `ZipArchive` repo (just current HEAD) to ensure things wouldn't break in future

Reply:
done